pkg/util/types.go

```diff
@@ -0,0 +1,33 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package util
+
+import "fmt"
+
+const (
+	// PodGroupLabel is the default label of batch scheduler
+	PodGroupLabel = "podgroup.scheduling.sigs.k8s.io"
```

Comment:
podgroup.scheduling.sigs.k8s.io --> pod-group.scheduling.sigs.k8s.io

Reply:
IMO， podgroup is alose OK. WDYT

Reply:
1. `pod-group` conforms to the naming standard of k8s. We talk about it before.
2. Hope to be consistent with the current implementation of the naming

Reply:
Ok, done!